include/zephyr/bluetooth/iso.h

```diff
@@ -217,6 +217,18 @@ struct bt_iso_recv_info {
 	uint8_t flags;
 };
 
+/** @brief ISO Meta Data structure for transmitted ISO packets. */
+struct bt_iso_transmit_info {
```

Comment:
Given the name of the function is `bt_iso_chan_get_tx_sync` suggest to rename this to
```suggestion
struct bt_iso_tx_info {
```

I wonder if we later will get an HCi command for RX info, or if we can implement that ourselves, in which case perhaps `bt_iso_link_info` would be a more future proof name? 

Although that is speculation, and I think it is fine to keep this as `tx_info`.

Reply:
I have no opinion on the name of this struct. The reason why I chose `transmit` instead of `tx` is to be somewhat aligned with the naming of the existing `bt_iso_recv_info` struct.